app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -491,6 +493,24 @@ class Contact extends React.Component {
                     <span className="font-weight-light">Has telephone but cannot use SMS or web-based reporting tool</span>
                   </div>
                 </Form.Group>
+                <Form.Group as={Col} lg={12}>
+                  <Form.Label className="input-label">
+                    INTERNATIONAL TELEPHONE NUMBER{schema?.fields?.international_telephone?._exclusive?.required && ' *'}
+                  </Form.Label>
+                  <Form.Control
+                    id="international_telephone"
+                    value={this.state.current.patient.international_telephone || ''}
+                    onChange={this.handleChange}
+                    size="lg"
+                    className="form-square"
+                    isInvalid={this.state.errors['international_telephone']}
+                  />
+                  <Form.Control.Feedback className="d-block" type="invalid">
+                    {this.state.errors['international_telephone']}
+                  </Form.Control.Feedback>
+                </Form.Group>
+                {this.state.current.patient.international_telephone &&
```

Comment:
what does this wanting banner look like just under the international phone field?  I feel like that's what we do across the board (keep it in the column) but not sure if that looked funny

Reply:
hmm it looks like we do have it span across the whole thing (ignore the weird padding for the int phone warning, was playing around with making it only span the col)

![Screen Shot 2021-09-22 at 2 51 22 PM](https://user-images.githubusercontent.com/9956561/134404137-d9cb384e-9e34-4ee7-815c-64810e27748b.png)

Reply:
In other spots in the enrollment carousel we don't though, so my thoughts are leaning towards making them both halves but would need to think about the best way to do it...  I think it does make sense since those warnings are associated one field in one column.  I can help with the styling if needed

Reply:
thoughts on something like this?

<img width="1746" alt="Screen Shot 2021-09-22 at 5 09 36 PM" src="https://user-images.githubusercontent.com/35042815/134422412-f447efd1-5811-4bb0-a9ff-89bde7433441.png">

Reply:
looks good to me!

Reply:
pushed: https://github.com/SaraAlert/SaraAlert/pull/1069/commits/c06e8e0c94a211c2d59e98b3ec09faad0e507f82